docs/api.yaml

```diff
@@ -814,7 +814,7 @@ tags:
     * `dataset.update` when a Dataset is updated.
     * `dataset.update.publish` when a Dataset is published.
     * `entity.create` when an Entity is created.
-    * `entity.create.error` when there is an error during entity creation process.
+    * `entity.error` when there is an error during entity creation process.
```

Comment:
list other entity actions in API (`entity.update.version`, `entity.update.resolve`, `entity.delete`) not super relevant to this PR